drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -107,6 +107,12 @@ const IID IID_IAudioClient3 = __uuidof(IAudioClient3);
 const IID IID_IAudioRenderClient = __uuidof(IAudioRenderClient);
 const IID IID_IAudioCaptureClient = __uuidof(IAudioCaptureClient);
 
+#define SAFE_RELEASE(memory)   \
+	if ((memory) != nullptr) { \
+		(memory)->Release();   \
+		(memory) = nullptr;    \
+	}
+
```

Comment:
Are you sure this is the proper way to fix the error?

Reply:
This works because `.Reset()` is failing when `ComPtr` is `NULL`.